install/kubernetes/cilium/values.schema.json

```diff
@@ -1502,6 +1502,18 @@
             "string"
           ]
         },
+        "enableConntrackLocal": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
```

Comment:
I appreciate the perspective, but I strongly disagree about adding this conntrack local Helm configuration option. The feature is known to be broken in various cases which is why we've deprecated it. We can't add new ways to try to use the feature when we're actively working to remove the feature.
```suggestion
```

Reply:
Didn't know that the feature was broken, sorry about that. I removed the options and updated the documentation.